### ISSUE TITLE
ClientSizeChanged behaviour differs from XNA

### ISSUE THREAD

**Issue by KakCAT** [MonoGame/MonoGame#7485]
ClientSizeChanged is triggered when a window is moved in MonoGame, but moving the window in XNA does not trigger it.

Failing code sample:
```
using System;

namespace XnaTest1
{
class GameTest : Game1
{

protected override void Initialize()
{
	base.Initialize();

	Window.AllowUserResizing = true;
	Window.ClientSizeChanged += Window_ClientSizeChanged;

}

private void Window_ClientSizeChanged(object sender, EventArgs e)
{
	System.Diagnostics.Debug.WriteLine ("Client Size Changed");
}


}
}
```

Latest MG development branch, Windows, WindowsDX

**Comment by iProgramMC** [comment]
I guess a workaround would be just to check if the new clientsize matches the old clientside, would be pretty useful to see when a window moved I suppose?

**Comment by KakCAT** [comment]
Yes, actually I had the client size check before realizing it was  a bug :)

I was reviewing my code and realized that a function named ClientSizeChanged required me to check if the client size had changed. First I thought I was doing something wrong, but then I tested that on XNA, which worked.

**Comment by iProgramMC** [comment]
I see, I see
